evap/contributor/templates/contributor_index.html

```diff
@@ -96,6 +96,10 @@
                                 {% else %}
                                     {% if course.avg_grade %}
                                         <div class="grade-bg text-center vertically-aligned" style="background-color: {{ course.avg_grade|gradecolor }}" data-toggle="tooltip" data-placement="top" title="{% trans 'Average grade' %}">{{ course.avg_grade|floatformat:1 }}</div>
+                                    {% else %}
+                                        <div class="grade-bg text-center vertically-aligned grade-bg-disabled" data-toggle="tooltip" data-placement="top" title="{% trans 'Not enough votes to publish results' %}">
+                                            <span class="far fa-eye-slash"></span>
+                                        </div>
```

Comment:
this seems like a functional change, what happened here previously?

Reply:
if the course was published, the grade was shown even if the thresholds have not been met.

Reply:
will be moved into the last commit